cuda_core/cuda/core/experimental/_memory/_device_memory_resource.pyx

```diff
@@ -66,63 +65,70 @@ cdef class DeviceMemoryResourceAttributes:
         self._mr_weakref = mr
         return self
 
-    @DMRA_mempool_attribute(bool)
+    cdef int _getattribute(self, cydriver.CUmemPool_attribute attr_enum, void* value) except?-1:
```

Comment:
The return type is made `int` so that we have a very simple error check `except?-1`. If we were to use `void` (which made sense back in Cython 0.29.x) then we would have to use `except*` here, which adds a slight overhead.